files/bridgesi/sprint-1-2_18inchPipePile_bridgeSI_Georef_A.ifc

```diff
@@ -0,0 +1,110 @@
+#37=IFCDIRECTION((0.,0.,1.));
+#38=IFCCARTESIANPOINT((0.,0.,0.));
+#39=IFCAXIS2PLACEMENT3D(#38,$,$);
+#40=IFCCOLOURRGB($,1.,0.,0.);
+#41=IFCSURFACESTYLERENDERING(#40,$,$,$,$,$,$,$,.BLINN.);
+#42=IFCSURFACESTYLE($,.POSITIVE.,(#41));
+#43=IFCPRESENTATIONSTYLEASSIGNMENT((#42));
+#44=IFCSTYLEDITEM(#36,(#43),$);
+#45=IFCSHAPEREPRESENTATION(#19,'Body','SweptSolid',(#36));
+#46=IFCPRODUCTDEFINITIONSHAPE($,$,(#45));
+#47=IFCLOCALPLACEMENT($,#48);
+#48=IFCAXIS2PLACEMENT3D(#38,#50,#49);
+#49=IFCDIRECTION((0.,1.,0.));
+#50=IFCDIRECTION((0.,0.,1.));
+#51=IFCRELCONTAINEDINSPATIALSTRUCTURE('1doFj9Xr98_w5SJX5JzQnT',$,$,$,(#28),#23);
+
+#54=IFCSOLIDSTRATUM('0ceaZL0r96X8U0KI9iYXWu',$,'Soil Slice #1 Subslice #1',$,$,#75,#74,$,$);
+#55=IFCARBITRARYPROFILEDEFWITHVOIDS(.AREA.,$,#56,());
+#56=IFCPOLYLINE((#57,#58,#59,#60,#57));
+#57=IFCCARTESIANPOINT((540.,540.));
+#58=IFCCARTESIANPOINT((-540.,540.));
+#59=IFCCARTESIANPOINT((-540.,-540.));
+#60=IFCCARTESIANPOINT((540.,-540.));
+#61=IFCEXTRUDEDAREASOLID(#55,#64,#62,5000.);
+#62=IFCDIRECTION((0.,0.,1.));
+#63=IFCCARTESIANPOINT((0.,0.,-600.));
+#64=IFCAXIS2PLACEMENT3D(#63,$,$);
+#65=IFCCOLOURRGB($,0.8666666666666667,0.8666666666666667,0.7019607843137254);
+#66=IFCCOLOURRGB($,0.8666666666666667,0.7019607843137254,0.7019607843137254);
+#67=IFCCOLOURRGB($,0.7019607843137254,0.7019607843137254,0.7019607843137254);
+#68=IFCCOLOURRGB($,0.6784313725490196,0.8470588235294118,0.9019607843137255);
+#69=IFCSURFACESTYLERENDERING(#65,0.7,$,$,$,$,$,$,.BLINN.);
+#70=IFCSURFACESTYLE($,.POSITIVE.,(#69));
+#71=IFCPRESENTATIONSTYLEASSIGNMENT((#70));
+#72=IFCSTYLEDITEM(#61,(#71),$);
+#73=IFCSHAPEREPRESENTATION(#19,'Body','SweptSolid',(#61));
+#74=IFCPRODUCTDEFINITIONSHAPE($,$,(#73));
+#75=IFCLOCALPLACEMENT($,#76);
+#76=IFCAXIS2PLACEMENT3D(#63,#78,#77);
+#77=IFCDIRECTION((1.,0.,0.));
+#78=IFCDIRECTION((0.,0.,1.));
+
+#79=IFCRELDEFINESBYPROPERTIES('2m3xQJuzPBv8g5XeS6QfpR',$,$,$,(#54),#80);
+#80=IFCPROPERTYSET('270I2FssLFE8jU$CnpxRCM',$,'SoilParameters',$,(#82,#83,#84));
+#82=IFCPROPERTYSINGLEVALUE('Internal Friction Angle',$,IFCTEXT('30'),$);
+#83=IFCPROPERTYSINGLEVALUE('Unit Weight',$,IFCTEXT('110'),$);
+#84=IFCPROPERTYSINGLEVALUE('Subgrade Modulus',$,IFCTEXT('30'),$);
+
+#85=IFCRELDEFINESBYPROPERTIES('2m3xQJuzPBv8g5XeS6QfpR',$,$,$,(#28),#86);
+#86=IFCPROPERTYSET('270I2FssLFE8jU$CnpxRCM',$,'SteelParameters',$,(#87,#88));
+#87=IFCPROPERTYSINGLEVALUE('Yield Stress',$,IFCTEXT('60'),$);
+#88=IFCPROPERTYSINGLEVALUE('Youngs Modulus',$,IFCTEXT('29000'),$);
+
+#89= IFCCLASSIFICATION('BSI',$,$,'Masterformat',$,'https://www.csiresources.org/standards/masterformat',$);
+#90= IFCRELASSOCIATESCLASSIFICATION('3CySecA9nEdRmCY328dhef',$,$,$,(#1),#89);
+
+#91 = IFCSITE('2q0A6$eDbEJx$bZGM8f7tv', $, 'Default Bridge', $, $, #24, $, $, .ELEMENT., (0, 0, 0), (0, 0, 0), 0., $, $);
+#92 = IFCRELAGGREGATES('2w4vtdhVoEH810nVKpCtY1', $, 'Project Bin', 'Project to Spatial Element', #1, (#91));
+
+#93= IFCPROJECTEDCRS('EPSG:12364',$,$,$,$,$,$);
```

Comment:
Out of curiosity: I can't find any reference to this EPSG code - can you provide a link to a source? (If you have simply made up the number, we encourage you to use a valid EPSG number, e.g. from https://epsg.io/ - the files are observed by domain experts as well :rocket:)